sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -90,9 +90,6 @@ describe("EventHub Receiver #RunnableInBrowser", function(): void {
   });
 
   describe("with EventPosition specified as", function(): void {
-    // TODO: Enable following test as part of https://github.com/Azure/azure-sdk-for-js/issues/3714
-    // After we ensure the multiple receiveBatch calls on the user facing receiver work as expected
-    /*
```

Comment:
Update this test to use `receiver.receiveBatch()` instead of using the batching receiver directly. That will include the fix that we are doing under the hood. Using batching receiver directly will result in the drain issue.

Create the issue once, use it multiple times

Reply:
Update the tests in this block to make a `receiveBatch()` call before sending the events. This should return 0 events. The `receiveBatch()` after sending the events should receive expected results. This way, we get test coverage for the fix you have done

Reply:
We still don't have any test cases to cover this feature. Please add a test such that
- create a receiver from last enqueued offset or sequence number
- receiveBatch(5) should get 0 results
- send 5 events
- receiveBatch(5) on the same receiever should get the 5 events

Reply:
Added the case to cover this scenario by creating a receiver from last enqueued sequence number.

Commented out the testcase #3709 (comment) mentioned in the issue because there is a issue when create receiver from latest().
Let's say:
- create a receiver from latest
- receiveBatch(5) gets 0 results
- send 5 events
- receiveBatch(5) on the same receiever also gets 0 because the checkpoint set to latest

Created an issue here https://github.com/Azure/azure-sdk-for-js/issues/3938 to track this.